config_scenario1.py

```diff
@@ -0,0 +1,64 @@
+"""Example configuration file
+
+This file aims to set parameters to realistic values
+"""
+
+import logging
+import itertools
+import numpy as np
+from DAS.shape import Shape
+
+dumpXML = 1
+visualization = 1
+logLevel = logging.INFO
+
+# number of parallel workers. -1: all cores; 1: sequential
+# for more details, see joblib.Parallel
+numJobs = -1
+
+# distribute rows/columns evenly between validators (True)
+# or generate it using local randomness (False)
+evenLineDistribution = False
+
+# Number of simulation runs with the same parameters for statistical relevance
+runs = range(1)
+
+# Number of validators
+numberNodes = [8000]
```

Comment:
Our crawler Armiarma currently shown about 11K beacon nodes in the network, I would suggest using that.